nidx/nidx_relation/tests/test_writer.rs

```diff
@@ -29,43 +33,50 @@ fn test_index_docs() -> anyhow::Result<()> {
 
     let resource = Resource {
         resource: Some(ResourceId {
-            uuid: "uuid".to_string(),
+            uuid: "01808bbd8e784552967a4fb0d8b6e584".to_string(),
```

Comment:
NIT: We could set a variable and clone it instead of repeating the string over and over